src/components/LessonActions/components/LessonAction/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {Icon} from 'egghead-ui'
+
+export default ({
```

Comment:
Thanks Ian. We actually started with egghead-ui button but decided to make this new component here; here is the history: https://github.com/eggheadio/egghead-instructor-center/issues/176